### ISSUE TITLE
Allow csv inputs and outputs

### ISSUE THREAD

**Issue by rchan26** [alan-turing-institute/prompto#102]
From some feedback, people have mentioned they would prefer if the output was a data frame or CSV file. In the case where a JSONL file is outputted, it seems people often convert this to a CSV anyway. 

To make this simpler, we can make some changes to allow for CSV inputs (which can get converted to JSONL / a list of prompt dictionaries) as well as allowing for CSV outputs:

Tasks:
- [x] Allow CSV inputs
  - [x] Implement function to convert CSV input to list of prompt dictionaries
  - [x] Make changes in CLI commands to allow for CSV inputs
- [x] Allow for CSV outputs
  - [x] After running an experiment, create a data frame for the results that can be stored in
  - [x] Implement function for outputting to csv using the data frame
- [x] Update tests

**Comment by rchan26** [comment]
For converting CSV to JSONL (and vice-versa), it seems easiest if every field/key is a column, e.g.
```
{"id": ..., "prompt": ..., "api": ..., "response": ..., "field1": ..., "field2"...}
```
gets converted to a CSV with these columns

**Comment by rchan26** [comment]
Implemented in #103
